src/UI/examples/Input/Field/TagInput/with_value.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Example show how to create and render a basic text input field with an error
+ * attached to it. This example does not contain any data processing.
+ */
+function with_value() {
+	//Step 0: Declare dependencies
+	global $DIC;
+	$ui = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	//Step 1: Define the text input field and attach some default value
+	$multi_select_input = $ui->input()->field()->tagInput("TagInput with Value")->withOptions([
+		6  => 'root',
+		13 => 'anonymous',
+		42 => 'fschmid',
+	])->withOptionsAreExtendable(true)->withSuggestionsStartAfter(1)->withValue([42]);
```

Comment:
If I remember our yesterdays discussion correctly, this will only send a list of strings back to the server, right? Maybe the example should show how the postprocessing of the inputs server side works.

I think this also might be another, much stronger, argument for splitting the component instead of having `withOptionsAreExtendable`. If I understand correctly, `withOptionsAreExtendable` changes the type of the content of the field from a dict to a list. If I attached a transformation to the field that expects one of the two, the code will silently break when I switch the content later on via `withOptionsAreExtendable`.